### PR TITLE
3.7 release prep work

### PR DESCRIPTION
**Pull request by pascalchevrel**

### CODE REVIEW

File: app/views/changelog.php [hunk 1]
```diff
@@ -1,3 +1,83 @@
+<?=$release_title('3.7')?>
+<h3>End user visible changes</h3>
+<ul>
+    <li>
+        <?=$relnotes('better')?>
+        The <a href="/?sourcelocale=en-US&amp;locale=fr&amp;repo=aurora&amp;t2t=t2t&amp;recherche=Bookmarks">glossary</a> view looks better and is visually more consistent with the rest of the site (flod)
+    </li>
+    <li>
+        <?=$relnotes('better')?>
+        The application no longer needs JavaScript to work (flod)
```
Comment (flodolo) [inline app/views/changelog.php]: I'd rephrase this: "The application is more usable when JavaScript is disabled". Some features still aren't available, but at least navigation and most parts work.

File: app/views/changelog.php [hunk 2]
```diff
@@ -1,3 +1,83 @@
...
+</ul>
+
+<h3>External API changes</h3>
+<ul>
+    <li>
+        <?=$relnotes('better')?>
+        <?=$issue(431)?>
+        HTML views which also exist as a JSON view now have a banner linking to the JSON file and our public API documentation (pascal)
+    </li>
+
```
Comment (flodolo) [inline app/views/changelog.php]: Extra blank line

File: app/views/changelog.php [hunk 3]
```diff
@@ -1,3 +1,83 @@
...
+        Repository now has a CONTRIBUTING.md file with code conventions displayed for each pull request (SkySymbol)
+    </li>
+    <li>
+        <?=$relnotes('new')?>
+        Added a command line option to glossaire.sh to not create a data snapshot when ran locally: <strong><code>glossaire.sh no-snapshot</code></strong> (flod)
+    </li>
+    <li>
+        <?=$relnotes('new')?>
+        <?=$issue(435)?>
+        Coding standards are enforced via php-cs-fixer,which is now installed via Composer. If you want to fix your code before committing, use the command <strong><code>./vendor/bin/php-cs-fixer fix</code></strong>. This is now a requirement as your build will fail on Travis CI if your code does not follow the project standards (pascal)
```
Comment (flodolo) [inline app/views/changelog.php]: space before which

**Comment by flodolo** [comment]
Looks good with nits. I wonder if we should create a rule for <code> in changelog.css

Example:
```
code {
    font-family: Courier, Courier New, monospace;
    font-weight: normal;
    background: #ddd;
    padding: 0 0.3em;
    white-space: nowrap;
}
```

**Comment by flodolo** [comment]
Result

schermata 2015-04-09 alle 17 48 57

**Comment by pascalchevrel** [comment]
I would prefer something less grayish, this is a merge of both our styles:
capture d ecran 2015-04-09 a 18 02 44

**Comment by flodolo** [comment]
What about #fff or even #fafafa? Full white is really bright :-)

**Comment by pascalchevrel** [comment]
#fafafa it is :)
